build.fsx

```diff
@@ -47,8 +47,17 @@ let gitHome = "https://github.com/" + gitOwner
 let gitName = "VisualFSharpPowerTools"
 let cloneUrl = "git@github.com:fsprojects/VisualFSharpPowerTools.git"
 
-// Ensure to use MSBuild 12 for the main solution
-setEnvironVar "MSBuild" (ProgramFilesX86 @@ @"\MSBuild\12.0\Bin\MSBuild.exe")
+// Ensure to use MSBuild 12 for the main solution if it exists, otherwise use version 14
+let msBuildVersionToTry = ["12.0";"14.0"]
```

Comment:
Good point, I was maybe looking to far ahead (up to msbuid 22 :)) will make it a simpler condition as you suggest.